metadata-ingestion/src/datahub/ingestion/source/looker/lookml_concept_context.py

```diff
@@ -413,3 +455,14 @@ def is_sql_based_derived_view_without_fields_case(self) -> bool:
             return True
 
         return False
+
+    def is_direct_sql_query_case(self) -> bool:
+        # pattern 7
+        # sqlglot doesn't have a function to validate whether text is valid SQL or not.
+        # Applying a simple logic to check if sql_table_name contains a sql.
+        # if sql_table_name contains sql then its value starts with "(" and checking if "select" is present in side the
```

Comment:
sql doesn't need to start with `(`

or is this something specific to lookml sql_table_name?

Reply:
specific to sql_table_name